source/sphinx_extensions/plugin_directory/templates/action.rst

```diff
@@ -1,6 +1,28 @@
 {{ title }}
 {{ '=' * title|length }}
 
+{% if has_citations %}
+.. raw:: html
+
+   <table class="table action-info">
+     <tbody>
+       <tr>
+         <th scope="row"><p></p>Citations</th>
```

Comment:
The extra `<p></p>` is just because the sphinx extension adds an empty paragraph before the bibliography that I couldn't get rid of. So this makes the vertical alignment work again.

Reply:
👍